docs/config.md

```diff
@@ -438,6 +438,22 @@ FF
 
 Note that the adapter does not automatically adapt the pressure name for solvers that account for [hydrostatic pressure effects](https://www.openfoam.com/documentation/guides/latest/doc/guide-applications-solvers-variable-transform-p-rgh.html). In these cases, you may want to set `nameP p_rgh` to couple `p_rgh`, as `p` is a derived quantity for these solvers.
 
+#### Restarting FSI simulations
+
+Restarting a coupled simulation using the OpenFOAM adapter works in principle in the same way as restarting a standalone OpenFOAM solver. However, the adapter and preCICE define the coupling interface and the interface node locations based on the mesh at the particular time. In case of FSI simulations, the interface deforms over time, which leads to the definition of a deformed interface during the restart. The boolean variable `restartFromDeformed` (`true` by default) allows to account for the previously accumulated interface deformation such that the initial interface configuration (`t = 0`) is completely recovered. The setting here needs to agree with the behavior of the selected solid solver.
+
+```c++
+FSI
+{
+    // Account for previous displacement during a restart
+    restartFromDeformed true;
+}
```

Comment:
The current documentation confused me for a while:

```suggestion
Interface1
{
    mesh                Fluid-Mesh;
    patches             (flap);
    
    // Account for previous displacement during a restart
    restartFromDeformed true;
}
```

Reply:
I moved the parameter now into the FSI dict (although it would fir conceptually better to the interface). I guess for the user, the FSI dict is the way to go.